internship-automation/pageobjects/myAccountPage.js

```diff
@@ -15,8 +15,8 @@ class MyAccountPage extends Page{
     }
 
     async enterPassword(password){
-        await this.inputUsername.click();
-        await this.inputUsername.setValue(password);
+        await this.inputPassword.click();
+        await this.inputPassword.setValue(password);
```

Comment:
Function name says enterPassword, but here you are clicking also. It probably means there should be another clickOnPasswordInput() :)